.tekton/pull-request.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: cachi2-pull-request
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
```

Comment:
Currently, the purpose of building an image on pull requests is just to verify that the build works. Is there some way to make those images temporary / clean them up later? Our current GH-actions setup does not pollute quay.io with PR builds

Reply:
There's a `quay.expires-after` label (https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/working_with_tags) but I don't see any way to set it dynamically from the docker-build pipeline. I can't think of other ways to do this.

Reply:
How about using build-args to pass tag to Containerfile and set expires-after LABEL for it in there? https://github.com/redhat-actions/buildah-build

Released images should have multiple tags so they should not be garbage collected.

Reply:
This Containerfile looks promising to pass git reference https://github.com/enterprise-contract/golden-container/blob/main/Containerfile#L3

Reply:
Well, unless we see value in using RHTAP scanners (i.e. Clair, ClamAV and Snyk) on PRs, we can probably just drop this `pull-request.yaml` and keep only the `push.yaml`.